src/main/java/club/sk1er/patcher/mixins/bugfixes/VertexBuffer_ResolveLockups.java

```diff
@@ -0,0 +1,23 @@
+package club.sk1er.patcher.mixins.bugfixes;
+
+import net.minecraft.client.renderer.vertex.VertexBuffer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.nio.ByteBuffer;
+
+@Mixin(VertexBuffer.class)
+public abstract class VertexBuffer_ResolveLockups {
```

Comment:
The mixin name should contain Mixin (`VertexBufferMixin_ResolveLockups`)

Reply:
Done